services/app-web/src/gqlHelpers/mutationWrappersForUserFriendlyErrors.ts

```diff
@@ -450,6 +450,11 @@ export const createRateQuestionWrapper = async (
                                                             'RateQuestionEdge',
                                                         node: {
                                                             ...newQuestion,
+                                                            createdAt: new Date(),
```

Comment:
Is this needed? If so, can we evenly apply - so add to contractquestionedge and then also should this be in the response cacheing as well?

Reply:
Good catch, I think I was getting failures in tests and modified this. It did fix the test, but I think the issue was the mock data. Removing these fields in this cache update has no effect on tests or actual rate question creation.